### DISCUSSION TITLE
Project 2 Part 4

### DISCUSSION THREAD

**Discussion by cgallagher1021** [cooperrc/computational-mechanics#159] [Q&A]
Good evening everyone! 
I was just beginning to catch my stride in this course when I came upon something that may be concerning. 
Is anyone getting an NYSE data line that looks more flat rather than a gradual increase? 
If this is not an issue I'll will continue on, thanks for any feedback!

**Comment by cooperrc** [comment]
Usually, this is a problem of scales, e.g.

you made a model where average growth is $0.001/day, but the actual data has average growth $0.1/day _or vice-versa_.

The result it either measured data or model looks constant compared to the other

**Comment by aliciasle** [comment]
I had the same issue.  Play around with the loc and scale to make the curve fit right.
The loc will move the curve left and right.
The scale will make your curve steeper or flatter.

Here is my code for GRMN for example:
```#Step 3

rng = default_rng(42)
N_models = 100
dprice_model = rng.normal(size = (len(garmin_data_pre_2014), N_models), loc = 0.0, scale = .7)

plt.hist(dprice, 50, density=True, label = 'NYSE data')
plt.plot(x, price_pdf)
plt.hist(dprice_model[:, 0], 50, density = True, 
         histtype = 'step', 
         linewidth = 3, label = 'model prediction 1')
plt.title('GARMN changes in price over 4 years\n'+
         'avg: \${:.2f} stdev: \${:.2f}'.format(mean_dprice, std_dprice))
plt.legend(); ```

**Comment by johnf329** [comment]
for mine, I set loc to equal my dprice mean, and scale was the dprice st.dev
